modules/tflite-micro/Kconfig

```diff
@@ -8,3 +9,24 @@ config TENSORFLOW_LITE_MICRO
 	bool "TensorFlow Lite Micro Support"
 	help
 	  This option enables the TensorFlow Lite Micro library.
+
+if CPU_CORTEX_M
+
+config OPTIMIZED_CMSIS_NN_KERNELS
```

Comment:
I think this option needs to be clearer about what it is.
Perhaps `TENSORFLOW_LITE_MICRO_CMSIS_NN_KERNELS`

Reply:
Sounds good to me

Reply:
Done